conformance/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/NegationRuleInterpreter.scala

```diff
@@ -29,6 +29,7 @@ import za.co.absa.enceladus.utils.validation.SchemaPathValidator
 
 case class NegationRuleInterpreter(rule: NegationConformanceRule) extends RuleInterpreter {
 
+  //noinspection ScalaStyle
```

Comment:
Why this broad switch off of the Scalastyle check? 😟

Reply:
I think I did something to my IDE 😑